WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageParentViewModel.kt

```diff
@@ -47,6 +51,12 @@ class PageParentViewModel
     private lateinit var site: SiteModel
     private var isStarted: Boolean = false
     private var page: PageModel? = null
+    private var _isSearchInUse = false
```

Comment:
We only use the `_` prefix when you have 2 instances of the same live data, one is private and mutable and the other one is public and immutable. I'd just use `isSearchInUse` here.